runtime/include/chpltypes.h

```diff
@@ -191,6 +191,13 @@ typedef int64_t chpl_bool64;
 typedef void (*chpl_fn_p)(void*); // function pointer for runtime ftable
 typedef int16_t chpl_fn_int_t;    // int type for ftable indexing
 
+// Function table names and information, for VisualDebug use
+typedef struct _chpl_fn_info {
+  const char *name;
+  int fileno;
+  int lineno;
+} chpl_fn_info;
+
```

Comment:
I'm surprised to see this in chpltypes.h. I think I'd prefer it to be in another header. Within the runtime, I don't think this type is generally necessary to be defined. In the generated code, the type definition will need to be #included somehow.
You could make  a new header for this purpose.